carto-package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "postgis",
+    "current_version": {
+        "requires": {
```

Comment:
I've seen some packages requiring `gdal` in the original issue, so Postgis also should [require](https://trac.osgeo.org/postgis/wiki/UsersWikiPostgreSQLPostGIS):
- gdal > 1.10 (for Raster). Recommended 2.2.
- geos > 3.4. Recommended 3.6 for now (3.7 soon)
- proj.4 . Not sure about minimum or recommended release. The newest the better
- json-c (GeoJSON). Not sure about minimum or recommended release. The newest the better
- protobuf-c (for MVT). Not sure about minimum or recommended release. The newest the better

In all cases the package should be compatible with any newer stable release (gdal 2.3, geos 3.7, proj.4 5.1 (great naming), etc) so `>` should be used.

Reply:
The intent of the `carto-package.json` is to specify CARTO components and their dependencies among them. This component is considered a leaf of the dependency tree (graph?) as defined by Infrastructure.

I believe the dependencies of postgis itself are managed in some other way (omnibus?).


Adding @skgsergio in case he wants to add/clarify anything.

Reply:
No worries, is good to have it defined if you feel is not too much work to maintain them (for example @javitonino is doing it for cartodb https://github.com/CartoDB/cartodb/pull/14232/files). We are not going, at least now, to parametrize those dependencies but doesn't hurt to have them there (in fact is good for checking if we are using old/not recommended versions, don't think only in the automatic dep resolution).

Reply:
OK, I'll add them